examples/README.md

```diff
@@ -159,6 +159,7 @@ Example | File | Description
 `startup_system` | [`ecs/startup_system.rs`](./ecs/startup_system.rs) | Demonstrates a startup system (one that runs once when the app starts up)
 `state` | [`ecs/state.rs`](./ecs/state.rs) | Illustrates how to use States to control transitioning from a Menu state to an InGame state
 `system_chaining` | [`ecs/system_chaining.rs`](./ecs/system_chaining.rs) | Chain two systems together, specifying a return type in a system (such as `Result`)
+`system_closures` | [`ecs/system_closures.rs`](./ecs/system_closures.rs) | Shows how to create systems out of closures, including how to pass arbitrary arguments via `move` or `Local`.
```

Comment:
Is this too wordy?

Reply:
You could probably just reduce it to `Shows how to create and execute systems from closures` (or something to that extent)